Library/Homebrew/load_path.rb

```diff
@@ -7,7 +7,20 @@
 
 $LOAD_PATH.push HOMEBREW_LIBRARY_PATH.to_s
 
+require "utils/gems"
```

Comment:
I'd be tempted to move this require to `brew.rb` instead. This file is also used by `rubocops.rb`.